charts/refarch-templates/templates/hpa.yaml

```
@@ -0,0 +1,39 @@
+---
+apiVersion: autoscaling/v2
+kind: HorizontalPodAutoscaler
+metadata:
+  name: {{ include "getFullname" $data }}
+  labels:
+    {{- include "getLabels" $data | trim | nindent 4 }}
+spec:
+  scaleTargetRef:
+    apiVersion: apps/v1
+    kind: Deployment
+    name: {{ $module.name }}
```

YAMLlint (1.35.1)

[error] too many spaces inside braces

(braces)